test/ArrayUtilsTest.php

```diff
@@ -10,7 +10,7 @@
 use Laminas\Stdlib\ArrayUtils\MergeReplaceKey;
 use Laminas\Stdlib\ArrayUtils\MergeReplaceKeyInterface;
 use Laminas\Stdlib\Exception\InvalidArgumentException;
-use Laminas\Stdlib\Parameters;
+use laminas\stdlib\parameters;
```

Comment:
Seems wrong :thinking: 

A lot of these changes in the test suite seem to be overall broken too

Reply:
Got all of these fixed with 50c8715